src/com/fedorowiat/menu/LoginMenu.java

```diff
@@ -15,13 +17,22 @@ public LoginMenu() {
         this.input = new Scanner(System.in);
     }
 
-    public void showLoginMenu() {
+    public void showLoginMenu(DataBase dataBase) {
         System.out.print("\n\nPodaj login: ");
-        String login = input.nextLine();
-
+        var login = input.nextLine();
         System.out.print("Podaj hasło: ");
-        String password = input.nextLine();
+        var password = input.nextLine();
         System.out.println("\n\n");
+            if(dataBase.checkLoginData(login, password)){
```

Comment:
tego ifa bym dal przed reszta pojedynczo, zeby od razu zwracal komunikat tak o: 
`if(!dataBase.checkLoginData(login, password)){
System.out.println("Błędny login lub hasło");
return;
} 
`
i dopiero reszta. Raczej unikamy zagniezdzonych if'ow bo ciezko sie to czyta